File: app/app/Model/Member/Response/MemberSearchResponse.swift
struct MemberSearchResponse: Codable, Identifiable {

    let memberId: Int64
    let profileUrl: String?
    let nickname: String
    let gender: String
    let age: Int
    let comment: String?
    let distance: Double?
    let likes: Int
    let updatedAt: String
    let similarityScore: Double

    var id: Int64 { memberId }
}

File: server/src/main/kotlin/com/pidulgi/server/member/dto/response/MemberSearchResponse.kt
package com.pidulgi.server.member.dto.response

import com.pidulgi.server.member.entity.type.Gender
import java.time.LocalDateTime

data class MemberSearchResponse(

    val memberId: Long,
    val profileUrl: String?,
    val nickname: String,
    val gender: Gender,
    val age: Int,
    val comment: String?,
    val distance: Double?,
    val likes: Int,
    val updatedAt: LocalDateTime,
    val similarityScore: Double,
)
